tests/unit/dataloader/test_tf_dataloader.py

```diff
@@ -20,18 +20,22 @@
 import time
 import timeit
 
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.metrics import roc_auc_score
+
 from merlin.core.dispatch import HAS_GPU, make_df
 from merlin.io import Dataset
 from merlin.schema import Tags
 
+pytestmark = pytest.mark.tensorflow
```

Comment:
I like adding this ability to mark the tests based on framework! .Can we do the same thing for the JAX/PT tests too - so that we're consistent?

Reply:
This came from the pattern used in Merlin Models for marking tests with the names of the frameworks. I've added one to each framework test file.